### PR TITLE
Create CI for k2-tools

### PR DESCRIPTION
**Pull request by blackdog0403**
Modified Dockerfile to add k2dryrun.sh to test new built k2-tools image.

Do not merge when   [Add terraform dryrun role on kraken.provider PR(samsung-cnct/k2#590)](https://github.com/samsung-cnct/k2/pull/590) is done.

Please give some opinions about this. It's good to include k2dryrun.sh inside of k2-tools docker image?

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,67 @@
+podTemplate(label: 'k2-tools', containers: [
+    containerTemplate(name: 'jnlp', image: 'quay.io/samsung_cnct/custom-jnlp:0.1', args: '${computer.jnlpmac} ${computer.name}'),
+    // containerTemplate(name: 'k2-tools', image: 'quay.io/samsung_cnct/k2-tools:latest', ttyEnabled: true, command: 'cat', alwaysPullImage: true, resourceRequestMemory: '1Gi', resourceLimitMemory: '1Gi'),
+    // containerTemplate(name: 'e2e-tester', image: 'quay.io/samsung_cnct/e2etester:0.2', ttyEnabled: true, command: 'cat', alwaysPullImage: true, resourceRequestMemory: '1Gi', resourceLimitMemory: '1Gi'),
```
Comment (coffeepac) [inline Jenkinsfile]: please remove the commented out lines

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  sh(command)
+}
+
+def kubectl(command) {
+  "kubectl exec -i ${env.HOSTNAME} -c ${env.CONTAINER_NAME} -- /bin/sh -c 'cd ${env.WORKSPACE} && ${command}'"
+}
+
+def dockerctl(command) {
+  ""
+}
```
Comment (coffeepac) [inline Jenkinsfile]: don't need this function

File: k2dryrun.sh [hunk 3]
```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+apk update
+apk add git
+git clone --branch master  https://github.com/samsung-cnct/k2.git ~/kraken
+cd ~/kraken
+ls -al
+build-scripts/fetch-credentials.sh
```
Comment (coffeepac) [inline k2dryrun.sh]: do you need to fetch the credentials?  there shouldn't be any remote requests made so if the files need to exist it is probably sufficient to `touch` the required files and leave them empty.

File: Jenkinsfile (hunk 1 above)
Comment (joejulian) [inline Jenkinsfile]: Please remove these lines if they are not needed.

File: Jenkinsfile [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+    }
+  }
+  sh(command)
+}
+
+def kubectl(command) {
+  "kubectl exec -i ${env.HOSTNAME} -c ${env.CONTAINER_NAME} -- /bin/sh -c 'cd ${env.WORKSPACE} && ${command}'"
+}
+
+def dockerctl(command) {
```
Comment (joejulian) [inline Jenkinsfile]: dockerctl is unused

**Pull request merged**